### DISCUSSION TITLE
Does PDIIIF allow you to kit multiple manifests together?

### DISCUSSION THREAD

**Discussion by brittnylapierre** [jbaiter/pdiiif#34] [Q&A]
For example if I have some pages from one manifest, and other pages from another, that I want to download into one PDF in one go. If not is this something that you're interested in implementing in the future, or if the mirador plugin made my Harvard supports something like this? Thanks!

**Comment by jbaiter** [comment]
I think you would be better served by a dedicated tool to join two PDFs together.

pdiiif is currently designed around the "one PDF per manifest" use case and supporting multiple manifests or IIIF collections is out of scope at the moment.

You could also try the [IIIF Manifest Editor](https://digital.bodleian.ox.ac.uk/manifest-editor/#/?_k=h1sf9o) to create a new manifest with your desired pages and then convert that to PDF.
